### PR TITLE
new template for organizing a Test Week

### PR DESCRIPTION
**Pull request by miabbott**

### CODE REVIEW

**Comment by miabbott** [comment]
@dustymabe WDYT?  Maybe it could just be in the `docs` folder instead?

File: .github/ISSUE_TEMPLATE/test-week.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
+---
+name: Schedule a Fedora Test Week
+about: Schedule a Fedora Test Week for a new Fedora major release
+title: ''
+labels: 'community', 'meeting'
+assignees: ''
+---
+
+## Initial Tasks
```
Comment (dustymabe) [inline .github/ISSUE_TEMPLATE/test-week.md:9]:
```suggestion
## Initial Tasks (to be done at least one week before test week)
```

File: .github/ISSUE_TEMPLATE/test-week.md [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+
+[ ] Open this ticket in the `fedora-coreos-tracker` repo
+[ ] Open a ticket with [Fedora QA](https://pagure.io/fedora-qa/issues). See [F36 example](https://pagure.io/fedora-qa/issue/695)
+
+To be done after the Fedora QA folks have taken action on the QA ticket:
+
+[ ] Confirm the Test Day page is created on the Fedora Wiki.  For example: <https://fedoraproject.org/wiki/Test_Day:Fedora_36_CoreOS_2022-04-04>
```
Comment (dustymabe) [inline .github/ISSUE_TEMPLATE/test-week.md:16]:
```suggestion
[ ] Confirm the Test Day page is created on the Fedora Wiki.
    - For example: https://fedoraproject.org/wiki/Test_Day:Fedora_36_CoreOS_2022-04-04
```

File: .github/ISSUE_TEMPLATE/test-week.md [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+[ ] Confirm the Test Day results app is created. For example: <https://testdays.fedoraproject.org/events/131>
```
Comment (dustymabe) [inline .github/ISSUE_TEMPLATE/test-week.md:17]:
```suggestion
[ ] Confirm the Test Day results app is created.
    - For example: https://testdays.fedoraproject.org/events/131
```

File: .github/ISSUE_TEMPLATE/test-week.md [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+[ ] Choose a day during the Test Week to host a video session for live debug help
+[ ] Setup a Google Meet or other video conference session
+[ ] Create a HackMD doc for capturing notes during live video session
+[ ] Find volunteers to enumerate new documentation + test cases requiired for Test Week
```
Comment (dustymabe) [inline .github/ISSUE_TEMPLATE/test-week.md:21]: should we make it clear this should be a dedicated video session before hand (i.e. a strategy session)

File: .github/ISSUE_TEMPLATE/test-week.md [hunk 5]
```diff
@@ -0,0 +1,48 @@
...
+[ ] File an issue on `fedora-coreos-tracker` with TODO items. For example: <https://github.com/coreos/fedora-coreos-tracker/issues/1147>
```
Comment (dustymabe) [inline .github/ISSUE_TEMPLATE/test-week.md:22]: Let's add another item here for requesting a fedora badge (like https://pagure.io/fedora-badges/issue/871)

File: .github/ISSUE_TEMPLATE/test-week.md [hunk 6]
```diff
@@ -0,0 +1,48 @@
...
+
+- Monitor `fedora-coreos-tracker` for new issues reported as part of Test Week
+- Monitor #fedora-coreos on IRC for new issues reported as part of Test Week
+- Ensure there is one or more representatives of Fedora CoreOS team present for live video session
+
+## After Test Week
+
+[ ] Update `fedora-coreos-tracker` ticket with any issues found
+[ ] Update `fedora-coreos-tracker` ticket with any documentation updates made
+[ ] Review Test Day results app and follow-up on any errors reported, if possible
```
Comment (dustymabe) [inline .github/ISSUE_TEMPLATE/test-week.md:48]: Add an item here for following up in the badges ticket with list of names of people who participated in test day.

**Review by dustymabe — COMMENTED** [review]
Thank you Micah for working on this!

File: .github/ISSUE_TEMPLATE/test-week.md (hunk 4 above)
Comment (miabbott) [reply .github/ISSUE_TEMPLATE/test-week.md:21]: sure thing!

**Comment by miabbott** [comment]
@dustymabe updated! :arrow_up:

**Review by dustymabe — APPROVED** [review]
LGTM

**Pull request merged**